plugin-server/tests/main/process-event.test.ts

```diff
@@ -1652,57 +1654,6 @@ test('long event name substr', async () => {
     expect(event.event?.length).toBe(200)
 })
 
-describe('validates eventUuid', () => {
```

Comment:
I moved these tests to plugin-server/tests/worker/ingestion/event-pipeline/populateTeamDataStep.test.ts

Reply:
👍 that's great, those tests are cleaner now